config/locales/en.yml

```diff
@@ -52,6 +52,187 @@ en:
+            referral_source:
+              blank: "Please tell us how you heard about us"
+        checklist_assignment:
+          attributes:
+            checklist_template_item:  
+              blank: "Checklist template item must exist"
+            match:  
+              blank: "Match must exist"
+        checklist_assignment_item:
+          attributes:
+            checklist_template:  
+              blank: "Checklist template must exist"
+            name:  
+              blank: "Name can't be blank"
+            expected_duration_days:
+              blank: "Expected duration days can't be blank"
+              not_a_number: "Expected duration days is not a number"
+        contact:
+          attributes:
+            name:  
+              blank: "Name can't be blank"
+            email:  
+              blank: "Email can't be blank"
+            message:  
+              blank: "Message can't be blank"
+        default_pet_task:
+          attributes:
+            organization:  
+              blank: "Organization must exist"
+            name:  
+              blank: "Name can't be blank"
+            description:  
+              blank: "Description can't be blank"
+        donation:
+          attributes:
+            amount:  
+              blank: "Amount can't be blank"
+            currency:  
+              blank: "Currency can't be blank"
+        location:
+          attributes:
+            country:  
+              blank: "Please enter a country"
+            city_town: 
+              blank: "Please enter a city or town"
+            province_state:  
+              blank: "Please enter a province or state"
+        match:
+          attributes:
+            organization:  
+              blank: "Organization must exist"
+            pet:  
+              blank: "Pet must exist"
+            adopter_account:  
+              blank: "Adopter account must exist"
+        organization_profile:
+          attributes:
+            location:  
+              blank: "Location must exist"
+            organization:  
+              blank: "Organization must exist"
+            email:  
+              invalid: "Email is invalid"
+        organization:
+          attributes:
+            profile:  
+              blank: "Profile must exist"
+        pet:
+          attributes:
+            organization:  
+              blank: "Organization must exist"
+            name: "Name can't be blank"
+            birth_date:  
+              blank: "Birth date can't be blank"
+            breed:  
+              blank: "Breed can't be blank"
+            sex:  
+              blank: "Sex can't be blank"
+            species:  
+              blank: "Species can't be blank"
+            placement_type: 
+              blank: "Placement type can't be blank"
+            weight_from: 
+              blank: "Weight From can't be blank"
+              not_a_number: "Weight From is not a number"
+            weight_to:
```

Comment:
There should be another message here related to the error where `weight_to` is less than `weight_from`. You can test in the UI by adding a lower number in the `weight_to` field compared to `weight_from` field.